extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/data/input/kafkainput/KafkaInputFormatTest.java

```diff
@@ -683,7 +700,8 @@ public void testMissingTimestampThrowsException() throws IOException
       while (iterator.hasNext()) {
         Throwable t = Assert.assertThrows(ParseException.class, () -> iterator.next());
         Assert.assertEquals(
-            "Timestamp[null] is unparseable! Event: {foo=x, kafka.newts.timestamp=1624492800000, kafka.newkey.key=sampleKey, root_baz=4, bar=null, kafka...",
+            "Timestamp[null] is unparseable! Event: {kafka.newtopic.topic=sample, foo=x, kafka.newts"
```

Comment:
Nit: Maybe break the line right before `kafka.newts.timestamp` or before `Event:`